src/Codeception/Module/Doctrine2.php

```diff
@@ -272,6 +271,30 @@ public function flushToDatabase()
         $this->em->flush();
     }
 
+    /**
+     * Performs $em->refresh() on every passed entity. This is useful in acceptance tests where
+     * entity can become invalid due to external changes (relative to entity manager used in tests).
+     *
+     * @param object|object[] $entities
+     */
+    public function haveEntitiesRefreshed($entities)
```

Comment:
It isn't mandatory to use `have` prefix, it fact there was [discussion](https://github.com/Codeception/Codeception/issues/4268) proposing to get rid of it completely and [documentation](https://codeception.com/docs/02-GettingStarted#The-Codeception-Syntax) says that `Actions start with a plain english verb, like “click” or “fill”.`
So I think that it would be better to name these methods `refreshEntities` and `clearEntities`.

Reply:
I agree, these are awkward. So, changing them to `refreshEntities` and `clearEntityManager` (I think this one is more correct as it does not work on individual entities, it rather changes entity manager's internal state).